app/src/main/java/com/example/pocketcloset/Fragments/OutfitsFragment.java

```diff
@@ -51,15 +55,94 @@ public static OutfitsFragment newInstance(String param1, String param2) {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (getArguments() != null) {
-            mParam1 = getArguments().getString(ARG_PARAM1);
-            mParam2 = getArguments().getString(ARG_PARAM2);
         }
     }
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_camera, container, false);
+        user = getArguments().getParcelable("userToFilterBy");
+        return inflater.inflate(R.layout.fragment_outfits, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        allOutfits = new ArrayList<>();
+        adapter = new OutfitsAdapter(getContext(), allOutfits);
+        rvOutfits = view.findViewById(R.id.rvOutfits);
+
+        rvOutfits.setAdapter(adapter);
+        // set the layout manager on the recycler view
+        rvOutfits.setLayoutManager(new LinearLayoutManager(getContext()));
+
+
+        GridLayoutManager gridLayoutManager = new GridLayoutManager(
+                getContext(),
+                2 /* numberOfColumns */);
+
+        rvOutfits.setLayoutManager(gridLayoutManager);
+
+
+        swipeContainer = view.findViewById(R.id.swipeContainer);
+        //Setup refresh listener which triggers new data loading
+        swipeContainer.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                adapter.clear();
+                queryOutfits();
+                adapter.notifyDataSetChanged();
```

Comment:
Generally speaking, I'd prefer to have the `notifyDataSetChanged` calls all be in the Adapter itself. This is normally done by calling it in the Adapter's method that gives it new data.